twilio/http/token_manager_initializer.py

```diff
@@ -0,0 +1,16 @@
+from twilio.http.token_manager import TokenManager
+
+
+class TokenManagerInitializer:
+
+    org_token_manager = None
+
+    @classmethod
+    def set_token_manager(cls, token_manager: TokenManager):
```

Comment:
Test if this class is used -- it its an initializer name it as a factory
Test the case of custom token manager

Reply:
This file is removed now. This is not used anywhere